libraries/core/src/main/kotlin/com/bumble/appyx/core/integrationpoint/requestcode/RequestCodeBasedEventStreamImpl.kt

```diff
@@ -35,23 +41,33 @@ abstract class RequestCodeBasedEventStreamImpl<T : RequestCodeBasedEvent>(
         }
     }
 
+    private fun flushResultWhenSubscribersAppear(flow: MutableSharedFlow<T>, event: T) {
```

Comment:
I think in a future PR we should investigate whether we need to remove flows from the events map in cases where a node is never recreated